src/main/com/example011/radek/v1/Mines.java

```diff
@@ -0,0 +1,146 @@
+package main.com.example011.radek.v1;
```

Comment:
V zdani je: Na jednu minu musi byt minimalne 10 policek .. to se neda vyhrat. :D

Reply:
Zmátlo mě tam to "Taky musis overit aby pocet min nebyl vic jak pocet poli a byla moznost mit aspon jedno pole pro oznaceni", což v tomto případě nemůže nastat, pokud se nepletu :D